torch/testing/_creation.py

```diff
@@ -103,27 +101,47 @@ def make_tensor(
                 [False, True]], device='cuda:0')
     """
 
-    def _modify_low_high(low, high, lowest, highest, default_low, default_high, dtype):
+    def modify_low_high(
```

Comment:
No need to make this "private", i.e. leading underscore, since this is a local function and cannot be called from anywhere else.